src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Menu/List.cshtml

```diff
@@ -5,7 +5,7 @@
 }
 
 <style asp-src="~/OrchardCore.AdminMenu/Styles/admin-menu.min.css" debug-src="~/OrchardCore.AdminMenu/Styles/admin-menu.css" at="Head" depends-on="jQuery-ui"></style>
-<script asp-src="~/OrchardCore.AdminMenu/Scripts/admin-menu.min.js" debug-src="~/OrchardCore.AdminMenu/Scripts/admin-menu.js" at="Foot" depends-on="jQuery-ui"></script>
```

Comment:
Why remove these and the others in this module?

Reply:
reverted